cmd/pebble/cmd_checks.go

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) 2021 Canonical Ltd
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License version 3 as
+// published by the Free Software Foundation.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package main
+
+import (
+	"fmt"
+
+	"github.com/jessevdk/go-flags"
+
+	"github.com/canonical/pebble/client"
+)
+
+type cmdChecks struct {
+	clientMixin
+	Level      string `long:"level"`
+	Positional struct {
+		Checks []string `positional-arg-name:"<check>"`
+	} `positional-args:"yes"`
+}
+
+var checksDescs = map[string]string{
+	"level": `Check level to filter for ("ready" checks are considered "alive")`,
```

Comment:
Rearranged this. Also, this was wrong anyway -- `pebble checks` and the `/v1/checks` API doesn't filter checks in this way (it did originally), it's only the `/v1/health` endpoint that does, because that's kind of an "aggregation" endpoint.